data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py

```diff
@@ -226,22 +227,24 @@ def add_console_logging(add_handler):
         required=True,
         help='Name of the table cotaining the cope survey mapping information')
     parser.add_argument(
-        '--mapping_dataset',
-        dest='mapping_dataset_id',
+        '-q',
+        '--deid_questionnaire_response_map_dataset',
         action='store',
-        help=('Dataset name for the dataset containing deid mapping tables.  '
-              'For example, _deid_map and _deid_questionnaire_response_id.'))
+        dest='deid_questionnaire_response_map_dataset',
+        help=
+        'Identifies the dataset containing the _deid_questionnaire_response_map lookup table',
+        required=True)
 
     ARGS = parser.parse_args()
     if not ARGS.mapping_dataset_id:
         parser.error("The deid mapping dataset is required to run this script.")
 
     # clean_engine.add_console_logging(ARGS.console_log)
     add_console_logging(ARGS.console_log)
-    version_task = COPESurveyVersionTask(ARGS.project_id, ARGS.dataset_id,
-                                         ARGS.sandbox_dataset_id,
-                                         ARGS.cope_survey_dataset_id,
-                                         ARGS.cope_survey_table)
+    version_task = COPESurveyVersionTask(
+        ARGS.project_id, ARGS.dataset_id, ARGS.sandbox_dataset_id,
```

Comment:
The arguments here after `ARGS.sandbox_dataset_id` are out of order due to the missing `table_namer` parameter in the 4th position. You probably either need to add a `table_namer` parameter, or if you don't have one, pass in the keyword arguments you need as key-value pairs.